pkg/controller/workflow/reason.go

```diff
@@ -12,7 +12,9 @@ const (
 
 // Atlas Project reasons
 const (
-	ProjectNotCreatedInAtlas ConditionReason = "ProjectNotCreatedInAtlas"
+	ProjectNotCreatedInAtlas   ConditionReason = "ProjectNotCreatedInAtlas"
+	ProjectIPAccessInvalid     ConditionReason = "ProjectIpAccessListInvalid"
```

Comment:
[q] Why not keep the Go-style casing in reasons?

Reply:
ah, you mean `ProjectIPAccessListInvalid` instead of `ProjectIpAccessListInvalid`? 
yes, makes sense 👍